examples/app-basic/README.md

```diff
@@ -81,15 +54,25 @@ export const MyComponent = () => {
 
 ---
 
-### Create a full-width page with no navigation bar
+### Create a full-width page
 
-You can hide the tab navigation bar and have access to the full view right to the Grafana sidebar by calling the `onNavChanged()` function with an `undefined` (as it is done in [utils.routing.ts#L25]).
+You can hide the tab navigation bar and have access to the full view right to the Grafana sidebar by changing the layout of the `<PluginPage>` component.
```

Comment:
Maybe missing something, but this sentence confuses me. Maybe something like `You can hide the left sidebar and get access to the full width of the page by changing the layout of the `<PluginPage>` component.` would make it more clear?

Reply:
Yea, that sounds much better, will update it, thanks mate! 👍

Reply:
Fixed in https://github.com/grafana/grafana-plugin-examples/pull/245/commits/6acad4876b2fd4af0e5eb525b4b7ddb7eb27609e.